prow/hook/events.go

```diff
@@ -33,9 +33,10 @@ func (s *Server) handleReviewEvent(re github.ReviewEvent) {
 		"url":      re.Review.HTMLURL,
 	})
 	l.Infof("Review %s.", re.Action)
-	for p, h := range s.Plugins.ReviewEventHandlers(re.PullRequest.Base.Repo.Owner.Login, re.PullRequest.Base.Repo.Name) {
+	org, repo := re.PullRequest.Base.Repo.Owner.Login, re.PullRequest.Base.Repo.Name
+	for p, h := range s.Plugins.ReviewEventHandlers(org, repo) {
 		go func(p string, h plugins.ReviewEventHandler) {
-			pc := s.Plugins.PluginClient
+			pc := s.pluginClientFor(org, repo)
```

Comment:
@spxtr let me know what you think -- if you think we should just keep the resolution strategy the same I will not try to work around it